src/app/main/main.component.js

```diff
@@ -53,7 +53,7 @@ function routingConfig($stateProvider, $locationProvider, $urlRouterProvider){
       template: '<your-giving></your-giving>'
     })
     .state('payment-methods', {
-      url: "/profile/payment-methods.html",
+      url: "/payment-methods.html",
```

Comment:
It was updated in this PR, your link points at master.
https://github.com/CruGlobal/give-web/blob/71b5c417e5b0470dceab4e42179aa46828fd76e9/src/app/profile/yourGiving/stopStartRecurringGifts/stopStartRecurringGifts.modal.component.js#L23